sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/keyvault/KeyVaultProperties.java

```diff
@@ -178,4 +179,11 @@ public static String getPropertyName(String normalizedName, Property property) {
             .filter(s -> !s.isEmpty())
             .collect(Collectors.joining(DELIMITER));
     }
+
+    public static String getPropertyNameFromAzureProperties(Property property) {
+        return Stream.of(AzureProperties.PREFIX, property.getName())
```

Comment:
Not all properties are supported in `AzureProperties`, like `case-sensitive-keys`.